partiql-ast/src/main/resources/partiql_ast.ion

```diff
@@ -458,18 +458,6 @@ expr::[
     as_type:  '.type',
   },
 
-  // PartiQL special form `CAN_CAST`
-  can_cast::{
-    value:    expr,
-    as_type:  '.type',
-  },
-
-  // PartiQL special form `CAN_LOSSLESS_CAST`
-  can_lossless_cast::{
-    value:    expr,
-    as_type:  '.type',
-  },
-
```

Comment:
(self-review): I'm not sure of a way we can mark these `partiql-ast` nodes as deprecated and then remove them in a subsequent release. In the PR's current state, it will remove the nodes and corresponding `partiql-ast` code within the `v1` branch.

Reply:
from offline discussion, PartiQL team will remove these partiql-ast nodes from the `v1` branch.